results/GritLM__GritLM-7B-noinstruct/13f00a0e36500c80ce12870ea513846a066004af/model_meta.json

```diff
@@ -0,0 +1 @@
+{"name": "GritLM/GritLM-7B", "revision": "13f00a0e36500c80ce12870ea513846a066004af", "release_date": "2024-02-15", "languages": ["eng_Latn", "fra_Latn", "deu_Latn", "ita_Latn", "spa_Latn"], "n_parameters": null, "memory_usage": null, "max_tokens": null, "embed_dim": null, "license": null, "open_source": true, "similarity_fn_name": null, "framework": [], "loader": "gritlm_loader"}
```

Comment:
```suggestion
{"name": "GritLM/GritLM-7B-no-instruct", "revision": "13f00a0e36500c80ce12870ea513846a066004af", "release_date": "2024-02-15", "languages": ["eng_Latn", "fra_Latn", "deu_Latn", "ita_Latn", "spa_Latn"], "n_parameters": null, "memory_usage": null, "max_tokens": null, "embed_dim": null, "license": null, "open_source": true, "similarity_fn_name": null, "framework": [], "loader": "gritlm_loader"}
```

Reply:
This is probably the quick fix, but we are discussing a more stable solution [here](https://github.com/embeddings-benchmark/mteb/issues/1211).